index.js

```diff
@@ -11,9 +13,21 @@ module.exports = function (opts, cb) {
     return next()
   })
 
+  server.get('/api/v1/annotations/:pkg', function (req, res, next) {
+    clientManager.annotationsForPageLoad(req.params.pkg, function (annotations) {
+      res.send(annotations)
+      return next()
+    })
+  })
+
   // start listening on port.
   server.listen(opts.port, opts.host, function () {
     cb(null, server)
     console.info(server.name + ' listening at ' + server.url)
   })
+
+  clientManager.load(function (err) {
```

Comment:
@bcoe So we load addons once on startup, and then add new ones dynamically while running?